.travis.yml

```diff
@@ -1,28 +1,32 @@
 language: python
+
 python:
 - '3.6'
 - '3.5'
 - '3.4'
 
 matrix:
   include:
-    - python: 3.6
-      sudo: required
-      dist: trusty
+    - python: '3.4'
       env:
-      - TOXENV=style
+      - DNF=yum OS=centos7
       before_install:
-      - sudo sed -i 's/trusty/yakkety/' /etc/apt/sources.list
-      - sudo rm -rf /etc/apt/sources.list.d/*
-      - sudo apt-get -qq update && sudo apt-get -qq install shellcheck
+      - sed -i 's@fedora@jsmigel/centos-epel@' Dockerfile
```

Comment:
The docker image `jsmigel/centos-epel` is better than other docker images `centos6` or `centos7`?
https://hub.docker.com/_/centos/